README.md

```diff
@@ -1,37 +1,103 @@
+# Testing MarkLogic
 
-marklogic-unit-test includes the [original Roxy unit test framework for MarkLogic](https://github.com/marklogic-community/roxy/wiki/Unit-Testing) and 
-provides a few new features:
+marklogic-unit-test is an [ml-gradle bundle](https://github.com/marklogic-community/ml-gradle/wiki/Bundles) that allows
+a project to test MarkLogic code.  With one import a project immediately has access to:
 
-1. A REST endpoint for listing and running unit tests
-1. A small Java library for integrating MarkLogic unit tests into existing test frameworks like JUnit
-1. The ability to depend on this module as a true third-party dependency via ml-gradle
+1. A framework for writing and running MarkLogic unit tests, including several built in assertion functions
+1. A UI for viewing and running unit tests entirely within MarkLogic
+1. A REST endpoint to run and report unit tests with other tools
 
-To try this out locally, check out the [ml-gradle example project](https://github.com/marklogic-community/ml-gradle/tree/dev/examples/unit-test-project). 
+Testing MarkLogic from a Java project is made easy with marklogic-junit:
+
+1. Write MarkLogic tests entirely from Java
+1. Easily integrate MarkLogic unit tests into your favorite Java testing frameworks
+
+# Start using marklogic-unit-test
+
+If you'd like to use marklogic-unit-test check out this 
+[ml-gradle example project](https://github.com/marklogic-community/ml-gradle/tree/dev/examples/unit-test-project). 
 You can use that project's build.gradle file as an example of how to use marklogic-unit-test in your own project.
 
-## Releasing
+### Add marklogic-unit-test to `build.gradle`
+
+```aidl
+buildscript {
+  repositories {
+    jcenter()
+    mavenLocal()
+  }
+  dependencies {
+    classpath "com.marklogic:marklogic-unit-test-client:1.0.beta"
+    classpath "com.marklogic:ml-gradle:3.14.0"
+  }
+}
+
+apply plugin: "com.marklogic.ml-gradle"
+
+repositories {
+  jcenter()
+}
+
+dependencies {
+  mlBundle "com.marklogic:marklogic-unit-test-modules:1.0.beta"
+}
+```
+
+### Add Test Properties to `gradle.properties`
+
+```aidl
+// Settings for any ml-gradle project
+mlHost=localhost    // Assuming local development
+mlAppName=my-app    // Application name, defaults to my-app
+mlRestPort=8003     // Application Port, defaults to 8003
+mlUsername=         // Username used to manage MarkLogic
+mlPassword=         // Password used to manage MarkLogic
+
+
+// Settings specific to marklogic-unit-test
+mlTestRestPort=8004 // Testing port, view and run tests from this port
+
+mlModulePaths=src/main/ml-modules,src/test/ml-modules  // Add test directory to ml-gradle configuration so tests are
+                                                       // loaded.  Only necessary on automated testing environments.
+
+```
+
+### Deploy tests using ml-gradle
+
+Now that the environment is configured to load tests and setup a test application servier its time to deploy everything.
+```aidl
+> gradlew mlDeploy
```

Comment:
Is it better to use `./gradlew` so that it works for people who don't have gradle installed globally?